### PR TITLE
create release script for cordova + misc cleanup

### PR DESCRIPTION
**Pull request by Kirkkt**

### CODE REVIEW

File: lint.sh [hunk 1]
```diff
@@ -1,26 +0,0 @@
-#!/bin/bash
```
Comment (Kirkkt) [inline lint.sh]: this lint file is currently useless. It was a copy-pasta from that in web sdk. Let's kill it for now and make a new one when we've got time in the future.

**Comment by Kirkkt** [comment]
@dmitrig01 

One last thing before making the release :)
1, removed lint.sh, please see inline for reasoning
2, created release.sh, similar to that in web sdk
3, updated package.json, as it turned out we do have https://www.npmjs.com/package/branch-cordova-sdk :)

**Comment by dmitrig01** [comment]
one thing we may want to do: somehow ensure that the submodule is at the correct revision, and/or changing the submodule to be the correct revision

**Comment by Kirkkt** [comment]
For everyday development, I can live with "manually" update submodules (I have an alias to update submodules along with `git pull`)

for release, I wrote something in release.sh.

Let me know if it suffices.

**Comment by dmitrig01** [comment]
in release.sh, i see something to make sure you're on master; however I want to make sure that the _submodule_ is on the right revision (like `git checkout $VERSION` or something)

File: release.sh [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+[ $# -eq 0 ] && { echo "Usage: $0 v1.0.0"; exit 1; }
+
+VERSION_NO_V=$(echo $1 | tr -d "\nv")
+VERSION="v"$VERSION_NO_V
+DATE=$(date "+%Y-%m-%d")
+
+echo "Releasing Branch Cordova SDK"
+
+check_git_branch() {
```
Comment (Kirkkt) [inline release.sh]: nice catch! please check out this line till line 35.

PTAL

**Comment by Kirkkt** [comment]
PTAL (not sure if inline comments will ping you) @dmitrig01

**Comment by Kirkkt** [comment]
friendly ping
@dmitrig01

**Comment by dmitrig01** [comment]
:+1:

**Comment by dmitrig01** [comment]
just have to remember to release both at the same time

**Pull request merged**